src/sentry/static/sentry/app/views/settings/components/settingsNavigation.jsx

```diff
@@ -22,23 +22,20 @@ class SettingsNavigation extends React.Component {
   componentDidMount() {
     let {organization} = this.props;
     if (!organization) return;
-    let features = organization.features;
 
-    //Experiment variant is already assigned - this logs the exposure i.e. when the user gets to the settings page
-    let treatment = features.includes('sso-paywall-experiment-treatment');
-    let control = features.includes('sso-paywall-experiment-control');
+    let exposed = organization.experiments.SSOPaywallExperiment;
```

Comment:
is this organization object always going to use the detailed serializer on the python side? if not certain, do something like `organization.experiments && organization.experiments.SSOPaywallExperiment`

Reply:
if you're sure let's remove the safety check. otherwise someone else reading the code would think it could be null. and generally it's anyway better to always have it than it be optional. not a blocker for this PR, you can do it in another one if you want